src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs

```diff
@@ -723,7 +724,7 @@ private void MakeLayout(int width, int maxLines = int.MaxValue)
 								1,
 								_addedSpacing = GetSpacingAdd(_paint),
 								_metrics,
-								true,
+								false,
```

Comment:
Let's validate the screenshot tests (I restarted a build so they get created properly after a recent fix).

Reply:
As expected, the change is showing in all the tests, could you make a comparison with the behavior found on winui? (To see if it's properly reflecting the original behavior)